frameit/lib/frameit/editor.rb

```diff
@@ -37,17 +42,31 @@ def prepare_image
 
     def store_result
       output_path = screenshot.path.gsub('.png', '_framed.png').gsub('.PNG', '_framed.png')
-      image.format "png"
-      image.write output_path
+      image.format("png")
+      image.write(output_path)
       UI.success "Added frame: '#{File.expand_path(output_path)}'"
     end
 
     # puts the screenshot into the frame
     def put_into_frame
+      # We have to rotate the screenshot, since the offset information is for portrait
+      # only. Instead of doing the calculations ourselves, it's much easier to let
+      # imagemagick do the hard lifting for landscape screenshots
+      unless self.screenshot.portrait?
+        frame.rotate(-90)
+        @image.rotate(-90)
+      end
+
       @image = frame.composite(image, "png") do |c|
         c.compose "Over"
         c.geometry offset['offset']
       end
+
+      # We have to revert the state to be landscape screenshots
```

Comment:
This comment is correct, those are landscape screenshots, which we only temporarly rotate to be portrait for the imagemagick math. Should I update the comments?